about.html

```diff
@@ -43,22 +43,22 @@
     document.getElementsByTagName( "head" )[0].appendChild( link );
   }
   document.addEventListener("DOMContentLoaded", function(event) {
-  	reloadTheme("about.html");
+  	reloadTheme("about");
```

Comment:
If you are going to change these then please also change to match in darkmode js file

Reply:
Assuming you mean lang.js file, since darkmode.js doesn't refer to those literals. (changed)